src/docs/010_manual/20_install.adoc

```diff
@@ -58,20 +72,15 @@ NOTE: `dtcw.bat` wraps the `dtcw.ps1` script and executes it in powershell. This
 
 NOTE: In order to support development for your project on both, Linux and Windows, it makes sense to download all three scripts to your project.
 
-NOTE: if you use `cmd.exe`, please follow the instructions for powershell in the manual. We will not further differentiate between both.
+NOTE: if you use `cmd.exe`, please refer and follow the instructions for powershell. We will not further differentiate between both. You should however replace `.ps1` with `.bat`.
 
 IMPORTANT: `dtcw` uses bash as shell and thus tries to find java from within bash. Chances are that when you use `dtcw` from another shell like zsh, you shell knows about the right java version but your bash doesn't. In that case, please switch to bash and make sure that the right java version is known. Afterwards, you can switch back to your shell.
 
-The wrapper helps you in several ways:
 
-- it ensures that everyone who uses your repository, uses the same version of docToolchain
-- it keeps all the technology out of your repository
-- it wraps all your commands and directs them to your local docToolchain installation
-- even more, it checks the way you have docToolchain installed and if it hasn't been installed, it will install it for you via direct download or docker
 
-== run a first command
+== Running your first command
 
-To get started, you can run `tasks` as a first command:
+To get started with docToolchain, you can run `tasks` as a first command:
 
```

Comment:
To get started with docToolchain, run 'tasks' as the first command:


**@Bennykillua Again, get rid of unnecessary words.**